app/controllers/goldencobra/sessions_controller.rb

```diff
@@ -54,5 +50,14 @@ def logout
 
     def register
     end
+
+    private
+
+    def usermodel
+      return false if params[:usermodel].blank?
+      return false unless %w(user visitor).include?(params[:usermodel].to_s.downcase)
```

Comment:
Das könnten aber auch andere Models sein in den einzelnen Apps. Jedes Model welches `devise` nutzt ist ja eigentlich erlaubt.
Eventuell sollte man eher prüfen ob das verwendete Model auch Devise unterstützt?

`params[:usermodel].constantize.included_modules.include?(Devise::Models::Authenticatable)`